pom.xml

```diff
@@ -537,6 +537,18 @@
             </dependency>
 
             <dependency>
+                <groupId>org.openjdk.jmh</groupId>
+                <artifactId>jmh-core</artifactId>
+                <version>1.14.1</version>
```

Comment:
I opened a PR there :)
https://github.com/airlift/airbase/pull/93

I think dependency of JMH [0.7.1](https://github.com/prestodb/presto/pull/6236/files#diff-c4c53a310a6871d8b4e34808d9ac0592L101) in `presto-record-decoder/pom.xml` can be removed. What do you think?